setup.py

```diff
@@ -22,7 +22,7 @@
     packages=find_packages(),
     install_requires=[
         "gym>=0.26",
-        "numpy>=1.20.0",
+        "numpy~=1.23",
```

Comment:
compatibility for windows (c.f. @cgokmen)

Reply:
Thanks. I hate to be a PITA here but can you freeze a ~= version for all of the dependencies here?